docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md

```diff
@@ -675,13 +671,15 @@ f1_keywords:
   - "CS8821"
   - "CS8822"
   - "CS8823"
+  - "CS8824"
+  - "CS8825"
   - "CS8830"
   - "CS8831"
   - "CS8846"
   - "CS8847"
   - "CS8848"
   - "CS8849"
-  - "CS8850"
```

Comment:
CS8850 was retired.
It was "A positional record must have a non-empty parameter list", but the compiler team decided to allow empty parameter list on records.